openpype/hosts/houdini/hooks/set_default_display_and_view.py

```diff
@@ -0,0 +1,49 @@
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+
+
+class SetDefaultDisplayView(PreLaunchHook):
+    """Set default view and default display for houdini via OpenColorIO.
+
+    Houdini's defaultDisplay and defaultView are set by
+    setting 'OCIO_ACTIVE_DISPLAYS' and 'OCIO_ACTIVE_VIEWS'
+    environment variables respectively.
+
+    More info: https://www.sidefx.com/docs/houdini/io/ocio.html#set-up
+    """
+
+    app_groups = {"houdini"}
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+
+        OCIO = self.launch_context.env.get("OCIO")
+
+        # This is a cheap way to skip this hook if either global color
+        # management or houdini color management was disabled because the
+        # OCIO var would be set by the global OCIOEnvHook
+        if not OCIO:
+            return
+
+        houdini_color_settings = \
```

Comment:
Could we use `from openpype.hosts.houdini.api.lib import get_houdini_color_settings` here?

Reply:
`openpype.hosts.houdini.api.lib` includes Houdini specific and is not meant to be run outside of Houdini. 
I think it can be solved by moving `import hou`  inside each function that uses it. 
this also includes `openpype.hosts.houdini.api.pipeline`.